src/room/VideoPreview.tsx

```diff
@@ -64,7 +69,13 @@ export function VideoPreview({ matrixInfo, onUserChoicesChanged }: Props) {
 
   // Create local media tracks.
   const [videoEnabled, setVideoEnabled] = useState<boolean>(true);
-  const [audioEnabled, setAudioEnabled] = useState<boolean>(true);
+  const [audioEnabled, setAudioEnabled] = useState<boolean>(!muteAudio);
+
+  useEffect(() => {
+    if (muteAudio) {
+      setAudioEnabled(false);
+    }
+  }, [muteAudio]);
```

Comment:
Are you sure we need this? As soon as `audioEnabled` is set to false the user settings should contain the correct value (passed over: `onUserChoicesChanged`)

Reply:
Sorry for the late review...

Reply:
I am not sure I follow here...

Reply:
I think i now understand better what this is for. It guarantees, that we update the AudioEnabled flag if the muteAudio prop changes. (if more than 8 participants have joined)
Imo we should only use the participant count when opening the lobby to not have a unexpected mute state change that could confuse the user. In other words: if the participant count exceeds 8 while the user is in the lobby should not change the button mute state.

Reply:
Yes, I guess that would make sense, yes - /me gets it done